src/components/collection/summary/SummaryHeader.tsx

```diff
@@ -188,24 +190,26 @@ const SummaryHeader: React.FC<SummaryHeaderProps> = ({
                     fontWeight: 'bold',
                   }}
                 >
-                  {String.fromCharCode(183)}
+                  |
                 </Divider>
```

Comment:
You can set the `Divider` to be vertical using [the `orientation` prop](https://mui.com/material-ui/api/divider/#Divider-prop-orientation)